reference/commands/creator/new.rst

```diff
@@ -87,22 +87,51 @@ Creates a new package recipe template with a 'conanfile.py' and optionally,
 
       $ conan new mypackage/1.0@myuser/stable -t -ciglg -ciglc -ciu https://api.bintray.com/conan/myuser/myrepo
 
-- Create files from a custom, predefined user template:
+- Create files from a custom, predefined user template recipe:
 
   .. code-block:: bash
 
       $ conan new mypackage/1.0 --template=myconanfile.py
 
 
-  Conan will look for ``templates/myconanfile.py`` in the Conan local cache. If an absolute path is given as argument, it will be used instead.
+  Conan will look for ``templates/command/new/myconanfile.py`` or ``templates/myconanfile.py`` in the Conan local cache. If an absolute path is given as argument, it will be used instead.
   These files can be installed and managed by :ref:`conan_config_install` command. The templates use Jinja syntax:
 
   .. code-block:: text
 
     class {{package_name}}Conan(ConanFile):
         name = "{{name}}"
         version = "{{version}}"
+        conan_version = "{{conan_version}}"
 
-  Where ``name`` and ``version`` placeholders are defined from the command line, and ``package_name`` is a *CamelCase*
-  variant of the name: any valid Conan package name like ``package_name``, ``package+name``, ``package.name`` or
-  ``package-name`` will be converted into a suitable name for a Python class, ``PackageName``.
\ No newline at end of file
+  Supported place-holders:
+  
+  - ``name`` and ``version``: defined from the command line.
+  - ``package_name``: a *CamelCase* variant of the name. Any valid Conan package name like ``package_name``, ``package+name``, ``package.name`` or
+    ``package-name`` will be converted into a suitable name for a Python class, ``PackageName``.
+  - ``conan_version``: a string containing current conan version, e.g. ``1.24.0``.
```

Comment:
```suggestion
  - ``conan_version``: an object that renders as the current Conan version, e.g. ``1.24.0``.
```

In the future this probably will be an actual `Version` object, people will want to check for the version in order to add some features (or not) to the generated files.

```python
class Recipe(ConanFile):
   {% if conan_version >= "1.26" %}
   toolchain = "cmake"
   {% endif %}
   ...
```